### ISSUE TITLE
An event display out of range bug

### ISSUE THREAD

**Issue by AEXLORD** [schedule-x/schedule-x#147]
hi, all

## Environment
- Browser: chrome
- Browser-sersion: 120.0.6099.129 (Official Build) (x86_64)
- Schedule-X version: 
{
    "@schedule-x/calendar": "^1.4.0",
    "@schedule-x/date-picker": "^1.4.1",
    "@schedule-x/drag-and-drop": "^1.4.1",
    "@schedule-x/event-modal": "^1.4.1",
    "@schedule-x/react": "^1.3.0",
    "@schedule-x/scroll-controller": "^1.4.1",
    "@schedule-x/theme-default": "^1.4.1",
}

## Describe the bug

when I create a event with time or only date, like this
````
          {
                id: uuidv4(),
                title: 'Event 3',
                start: '2024-01-10 08:00',
                end: '2024-01-10 09:00',
                calendarId: 'work'
            }
````
I wish I could see it in the calendar like google calendar

<img width="326" alt="image" src="https://github.com/schedule-x/schedule-x/assets/23465011/fda39763-5b17-401d-9d89-9c0d198203c5">

but it shows like this :

<img width="416" alt="image" src="https://github.com/schedule-x/schedule-x/assets/23465011/df8e114d-82ef-4fed-a1f1-b1c512c3b781">

 and
 
<img width="589" alt="image" src="https://github.com/schedule-x/schedule-x/assets/23465011/a06e8de1-0322-4953-8df7-51836bace49b">

**Comment by tomosterlund** [comment]
Hi @AEXLORD. Thanks for opening this issue!

It looks to me like your application does not set all elements to `box-sizing: border-box`, which is the standardized box-sizing in most frontends. In other words, most applications have something like this in a global CSS file:

```css
* {
  box-sizing: border-box;
}
```

Until now I have just assumed everyone works like that, but maybe it would make sense if I write a rule for all child elements of the calendar, setting the box-sizing value explicitly.

**Comment by AEXLORD** [comment]
hi @tomosterlund , thank you for your answer. 
For a web app, defining global CSS can solve almost all formatting problems. But I believe that as a product, there should not be particularly big problems( maybe your test cases not include this?). 
As for users who want to customize their shapes in the future, they can achieve it by writing their own CSS, just like the callback functions you provide, which is very good.

At last, wishing you all the best in your work and good health

**Comment by tomosterlund** [comment]
Thank you so much for your kind words 😊

I just released a patch version 1.5.1 where this is now fixed.
